### PR TITLE
add image to jobs

### PR DESCRIPTION
**Pull request by sseerrggii**

### CODE REVIEW

File: jobs.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
 ---
 
 <div id="main-blog">
+  <img src=" /assets/img/jobs/coopdevs-meeting.jpg" class="rounded mx-auto d-block img-fluid" alt="Coopdevs Meeting">
```
Comment (enricostano) [inline jobs.html]: Useless space after `src="`

Comment (sseerrggii) [reply jobs.html]: true!

**Review by enricostano — APPROVED** [review]

**Comment by sauloperez** [comment]
Could you please share a screenshot to see how it looks like?

**Comment by sseerrggii** [comment]
fireshot capture 169 - coopdevs - http___127 0 0 1_4000_jobs html

First iteration :sweat_smile:

**Review by danypr92 — APPROVED** [review]

**Comment by sauloperez** [comment]
Is it too much to ask to add some bottom margin :sweat_smile: ?

**Comment by sseerrggii** [comment]
:innocent: :innocent:

fireshot capture 173 - coopdevs - http___127 0 0 1_4000_jobs html

**Review by sauloperez — APPROVED** [review]

**Review by enricostano — APPROVED** [review]

**Pull request merged**